greeter_plugin/greeter_summary.py

```diff
@@ -55,7 +55,8 @@ def op(name,
   summary_metadata = tf.SummaryMetadata()
   # We could put additional metadata other than the PLUGIN_NAME,
   # but we don't need any metadata for this simple example.
-  summary_metadata.plugin_data.add(plugin_name=PLUGIN_NAME, content="")
```

Comment:
nit: Could you instead please use 

```
tf.SummaryMetadata(
        display_name=display_name,
        summary_description=description,
        plugin_data=tf.SummaryMetadata.PluginData(
            plugin_name=PLUGIN_NAME,
            content=""))
```

and then remove the `display_name` and `summary_description` parameters from the call to `tf.summary.tensor_summary`? Same as the other change below.